spec/03-plugins/40-ai-response-transformer/01-transformer_spec.lua

```diff
@@ -55,98 +57,67 @@ local EXPECTED_RESULT = {
 }
 
 local SYSTEM_PROMPT = "You are a mathematician. "
-                   .. "Multiply all numbers in my JSON request, by 2. Return me this message: "
-                   .. "{\"status\": 400, \"headers: {\"content-type\": \"application/xml\"}, \"body\": \"OUTPUT\"} "
-                   .. "where 'OUTPUT' is the result but transformed into XML format."
-
-
-local client
-
-
-for _, strategy in helpers.all_strategies() do if strategy ~= "cassandra" then
-
-  describe(PLUGIN_NAME .. ": (unit)", function()
-
-    lazy_setup(function()
-      -- set up provider fixtures
-      local fixtures = {
-        http_mock = {},
-      }
-
-      fixtures.http_mock.openai = [[
-        server {
-            server_name llm;
-            listen ]]..MOCK_PORT..[[;
-            
-            default_type 'application/json';
-
-            location ~/instructions {
-              content_by_lua_block {
-                local pl_file = require "pl.file"
-                ngx.print(pl_file.read("spec/fixtures/ai-proxy/openai/request-transformer/response-with-instructions.json"))
-              }
-            }
-        }
-      ]]
-
-      -- start kong
-      assert(helpers.start_kong({
-        -- set the strategy
-        database   = strategy,
-        -- use the custom test template to create a local mock server
-        nginx_conf = "spec/fixtures/custom_nginx.template",
-        -- make sure our plugin gets loaded
-        plugins = "bundled," .. PLUGIN_NAME,
-        -- write & load declarative config, only if 'strategy=off'
-        declarative_config = strategy == "off" and helpers.make_yaml_file() or nil,
-      }, nil, nil, fixtures))
-    end)
-    
-    lazy_teardown(function()
-      helpers.stop_kong(nil, true)
-    end)
-
-    before_each(function()
-      client = helpers.proxy_client()
-    end)
+    .. "Multiply all numbers in my JSON request, by 2. Return me this message: "
+    .. "{\"status\": 400, \"headers: {\"content-type\": \"application/xml\"}, \"body\": \"OUTPUT\"} "
+    .. "where 'OUTPUT' is the result but transformed into XML format."
+
+
+describe(PLUGIN_NAME .. ": (unit)", function()
+  local mock
+  local mock_response_file = pl_path.abspath(
```

Comment:
Just a note to myself: perhaps we could add a new API to use a file as the response for `http_mock`.